tasks/gen/templates/model/operations/save_{{underscored_name}}.cr.ecr

```diff
@@ -1,2 +1,5 @@
 class Save<%= @name %> < <%= @name %>::SaveOperation
+<%- if @columns.any? -%>
+  permit_columns <%= @columns.map(&.name).join(", ") %>
```

Comment:
I am worried about security here. If someone generates a model with `admin` or `encrypted_password` or something, we wouldn't want to permit those. I think since we aren't generating forms and pages, it is ok to leave this out and users can add them depending on what they need. Also they may be creating things without params, in which case they don't need to permit_columns: `SaveUser.create(name: "John", email: "John@gmail.com")

One other idea would be to include this but comment it out and add an explaining comment:

```
# In order for clients to save to a column, you must permit them:
# permit_columns ....
```

Thoughts? cc @jwoertink @edwardloveall

Reply:
Yeah, that makes sense. You don't want to generate leaky code and have the user not know. If we don't permit them, and they try to use it, they will be told right away. Then if those fields are there and just commented out with a note, that makes it even easier because they don't need to look up docs on what to do.

Reply:
Right on. We could also link to the docs on it! https://luckyframework.org/guides/database/validating-saving#perma-permitting-columns

I'm thinking the comment can be included either way, and the commented out `permit_columns` can be generated if there are columns passed to the generator.

**If no columns provided**

```crystal
# To save user provided params to a column, you must permit them
# https://luckyframework.org/guides/database/validating-saving#perma-permitting-columns
#
# permit_columns column_1, column_2
```

*If columns are provided**

```crystal
# To save user provided params to a column, you must permit them
# https://luckyframework.org/guides/database/validating-saving#perma-permitting-columns
#
# permit_columns {{ the ones passed in from the generator }}
```

I'm guessing this could be cleaned up by adding a `columns_list` method to the `ModelTemplate` that is something like: 

```crystal
example_columns = [
  ColumnDefinition.new("column_1", ""), 
  ColumnDefinition.new("column_2", "")
]
(columns.any? ? columns : example_columns).map(&.name).join(", ")

Reply:
Makes a lot of sense! I'll attend to it tomorrow.

Reply:
Good morning! That's done now in both the model and the resource generators.